modules/RTC/ScreenObtainer.js

```diff
@@ -105,7 +105,7 @@ const ScreenObtainer = {
             channelCount: 2,
             echoCancellation: false,
             noiseSuppression: false
-        } : true;
```

Comment:
This change looka bogus. It would break the non-electron case. In Electron's case we are only calling this function when we know audio is going to be shared.